client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -148,6 +151,9 @@
     @Option(names = "--output-format", paramLabel = "<format>", defaultValue = "CSV", description = "Output format for batch mode [${COMPLETION-CANDIDATES}] " + DEFAULT_VALUE)
     public OutputFormat outputFormat;
 
+    @Option(names = "--pager", paramLabel = "<pager>", description = "Path to the pager program used to display the query results")
```

Comment:
It loosk like `System.getenv("TRINO_PAGER"))` is default, can reference to it?

Reply:
Done!